services/web/server/src/simcore_service_webserver/application_settings_utils.py

```diff
@@ -75,6 +75,9 @@ def convert_to_app_config(app_settings: ApplicationSettings) -> dict[str, Any]:
                 "enabled": app_settings.WEBSERVER_REDIS is not None,
                 "host": getattr(app_settings.WEBSERVER_REDIS, "REDIS_HOST", None),
                 "port": getattr(app_settings.WEBSERVER_REDIS, "REDIS_PORT", None),
+                "password": getattr(
+                    app_settings.WEBSERVER_REDIS, "REDIS_PASSWORD", None
```

Comment:
i guess you might also need here to `get_secret_valu3` ... although probably this part of the code is not anymore used. I think it was deprecated.